### ISSUE TITLE
Requires approvals before merging

### ISSUE THREAD

**Issue by LitoMore** [simple-icons/simple-icons#12383]
![](https://github.com/user-attachments/assets/6a64539e-ea19-49c7-872b-286fc367c1fa)

Our current `Require a pull request before merging` is off due to our release action using a direct push ([push to the develop branch](https://github.com/simple-icons/simple-icons/commit/3ab90a5c4e47fd3ad225c6a1678d2d6e564ffbde)). We need to find a workaround for it. 

Related discussion: https://github.com/orgs/community/discussions/25305

**Comment by LitoMore** [comment]
My suggestion is to drop the `Bump version` commit step. We can remove `slugs.md` and `version` fields from the develop branch. Then use a script to generate them before during the release PR (like our `reformat-markdown` does).

**Comment by mondeja** [comment]
> My suggestion is to drop the Bump version commit step.

Agree, note that this would also reduce the number of contributions of simple-icons[bot] to the half.

> We can remove slugs.md and version fields from the develop branch.

What "fields" are you referring?

**Comment by LitoMore** [comment]
> What "fields" are you referring?

The `version` in `package.json`. We can remove it from the `package.json` in the develop branch.

**Comment by mondeja** [comment]
Looks promising.

> For `update-cdn-link` similar scripts, those contents are only updated when preparing `npm publish`.

Take into account the tag too, which conforms the file tree published to Packagist.
